assets/src/scripts/components/hydrograph/graph-controls.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Hydrograph controls module.
+ */
+import { Actions } from '../../store';
+import { dispatch, link } from '../../lib/redux';
+import { audibleUI } from './audible';
+import { getCurrentVariableMedianStatistics } from '../../selectors/median-statistics-selector';
+import { isVisibleSelector, currentVariableTimeSeriesSelector } from './time-series';
+
+/*
+ * Create the show audible toggle, last year toggle, and median toggle for the time series graph.
+ * @param {Object} elem - D3 selection
+ */
+export const graphControls = function(elem) {
+
+    const graphControlDiv = elem.append('ul')
+        .classed('usa-fieldset', true)
+        .classed('usa-list--unstyled', true)
+        .classed('graph-controls-container', true);
+
+    graphControlDiv.append('li')
+        .call(audibleUI);
```

Comment:
I had forgotten about the audibleUI. This is another reason I would like this module to separate out the creating of the compare and median toggles. But in the interest of getting something that works, this is fine.

Reply:
I agree, I will look at splitting this logic out if there's time at the end of the sprint.  If I don't make it, I'll make a jira task.